posthog/src/test/java/com/posthog/java/AppTest.java

```diff
@@ -0,0 +1,57 @@
+package com.posthog.java;
+
+import java.util.HashMap;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Test;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest {
+
+    /**
+     * Would like this to be a separate example file, but couldn't figure out how to
+     * make that work yet
+     */
+    @Test
+    public void example() {
```

Comment:
Wanted to make an example file similar to how we have https://github.com/PostHog/posthog-python/blob/master/example.py but I couldn't quite figure out how to get project management tools / dependencies to be my friends.